docs/reference/remap/functions/redact.cue

```diff
@@ -0,0 +1,94 @@
+package metadata
+
+remap: functions: redact: {
+	category: "String"
+	description: """
+		Redact potentially sensitive data in `value` such as:
+
+		- credit card numbers
+		- custom patterns
+		- (more to come!)
+
+		This can act as a last resort in the case your applications log sensitive data by accident to ensure you do not
+		leak this information into your log storage.
```

Comment:
```suggestion
		Redact sensitive data in `value` such as:

		- credit card numbers
		- social security numbers
		- email addresses
		- and other forms of personally identifiable information via custom patterns

		This can help achieve compliance by ensuring sensitive data never leaves your network.
```

Reply:
This function doesn't currently have patterns for redacting social security numbers or email addresses, but those would be good additions. I plan to expand the pattern support after this initial PR is merged.

I took the other changes though 👍